### ISSUE TITLE
Misplaced rendering of subscript and exponent

### ISSUE THREAD

**Issue by bruinspaw** [njpipeorgan/wolfram-language-notebook#8]
Misplaced rendering of subscript and exponent while run the following code:

```
n = 2;
A = Table[Subscript[a, i], {i, 0, n}]; (* coefficient vector *)
X = Table[x^i, {i, 0, n}]; (* power vector *)
Solve[A.X == 0, x] 
```

The output of vs code:

<img width="980" alt="Screenshot 2021-12-30 下午10 07 32" src="https://user-images.githubusercontent.com/7016270/147759773-12192d41-3023-4846-8a12-624a8bc9d388.png">

The output of jupyter notebook(correct):

<img width="1103" alt="Screenshot 2021-12-30 下午10 08 35" src="https://user-images.githubusercontent.com/7016270/147759872-016cc9a2-5b3d-46fe-8b64-2c13c54d4024.png">

**Comment by njpipeorgan** [comment]
Thanks! I will get it fixed in the next version.

**Comment by njpipeorgan** [comment]
It has been fixed in v0.0.7.
